realm/realm-library/src/objectServer/java/io/realm/RealmFileUserStore.java

```diff
@@ -54,8 +54,20 @@ public SyncUser get() {
      * {@inheritDoc}
      */
     @Override
-    public void remove() {
-        nativeLogoutCurrentUser();
+    public SyncUser get(String identity) {
+        String userJson = nativeGetUser(identity);
+        if (userJson != null) {
+            return SyncUser.fromJson(userJson);
+        }
+        return null;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void remove(String identity) {
+        nativeLogoutUser(identity);
```

Comment:
I'm not sure we want to log the user out when removing it? Those two operations don't seem connected to me? Most likely the binding will log the user out before removing it, but I don't it as the UserStores responsibility?